scheduler/scheduler.go

```diff
@@ -1,15 +1,31 @@
 package main
 
 import (
-	"fmt"
+	"context"
+	"log"
 	"net/http"
+
+	worker "github.com/vs-labs/distributed-task-scheduler/worker/job"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
 )
 
 func main() {
 	http.HandleFunc("/hello", helloHandler)
 	http.ListenAndServe(":8081", nil)
+
 }
 
 func helloHandler(w http.ResponseWriter, r *http.Request) {
-	fmt.Fprint(w, "Hello, World!")
+	var conn *grpc.ClientConn
+	conn, err := grpc.Dial(":8080", grpc.WithTransportCredentials(insecure.NewCredentials()))
```

Comment:
Externalize port maybe?

Reply:
i would do that when we add make files and env setup

Reply:
since it should not be port but url anyhow

Reply:
Sounds good